ports/xmlsec/CMakeLists.txt

```
@@ -60,12 +60,10 @@ set(CMAKE_STATIC_LIBRARY_PREFIX "lib")
 add_library(xmlsec1 ${SOURCESXMLSEC})
 add_library(xmlsec1-openssl ${SOURCESXMLSECOPENSSL})
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/include include ${LIBXML2_INCLUDE_DIRS})
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/include include)
```

The exported targets still lack interface include directories. Untested:
```suggestion
include_directories(${CMAKE_CURRENT_BINARY_DIR}/include include)
target_include_directories(xmlsec1 PUBLIC
    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
    include
)
```

---

Thank you, fixed